shared/templates/template_OVAL_yamlfile_value

```diff
@@ -0,0 +1,58 @@
+{{% if target_oval_version >= [5, 11] %}}
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    <metadata>
+      <title>Check the value(s) of '{{{ YAMLPATH }}}' in the file "{{{ FILE }}}" (located in "{{{ PATH }}}")</title>
+      {{{- oval_affected(products) }}}
+      <description>The file "{{{ FILE }}}" in "{{{ PATH }}}" should {{% if NEGATE %}}not {{% endif %}}contain value "{{{ VALUE }}}" of type {{{ TYPE }}} at '{{{ YAMLPATH }}}'.</description>
+    </metadata>
+    <criteria>
+        <criterion comment="Make sure that {{% if NEGATE %}}no {{% endif %}}{{% if CHECK %}}({{{ CHECK }}}) {{% endif %}} element(s) of '{{{ YAMLPATH }}}' contain value '{{{ VALUE }}}'' of type '{{{ TYPE }}}''."{{% if NEGATE %}} NEGATE="true"{{% endif %}} test_ref="test_{{{ rule_id }}}"/>
+        {{% if NEGATE %}}
+        <criterion comment="Make sure that there is the file '{{{ FILE }}}' exists in '{{{ PATH }}}'." test_ref="test_file_for_{{{ rule_id }}}"/>
+        {{% endif %}}
+    </criteria>
+  </definition>
+
+  <local_variable id="{{{ rule_id }}}_file_location" datatype="string" comment="The actual path of the file to scan." version="1">
+      {{% if OCP_DATA %}}
+      <concat>
+        <variable_component var_ref="ocp_data_root"/>
+        <literal_component>{{{ PATH }}}</literal_component>
+      </concat>
+      {{% else %}}
+      <literal_component>{{{ PATH }}}</literal_component>
+      {{% endif %}}
+  </local_variable>
+
+  <ind:yamlfilecontent_test id="test_{{{ rule_id }}}" check="only one" comment="Find only one '{{{ FILE }}}' with '{{{ VALUE }}}' at '{{{ YAMLPATH }}}'." version="1">
```

Comment:
@evgenyz You are correct, only one object will be returned.
Reference, https://github.com/OpenSCAP/openscap/pull/1480#discussion_r382595294
In this case, `check="only one"` can be omited.

If still you'd like to make it clear that only one object is expected, the attribute that should be there is `check_existence="only_one_exists"`.
https://github.com/OVALProject/Language/blob/master/docs/oval-common-schema.md#---existenceenumeration---

Reply:
`OpenSCAP Error: File '/home/ekolesni/Devel/content/build/ssg-ocp4-ds.xml' line 26835: Element '{http://oval.mitre.org/XMLSchema/oval-definitions-5#independent}yamlfilecontent_test': The attribute 'check' is required but missing.
 [/builddir/build/BUILD/openscap-1.3.3/src/XCCDF/xccdf_session.c:613]`